packages/ui/src/Button.module.scss

```diff
@@ -128,107 +122,9 @@
-
-      &.warningSecondary {
-        border: $borderWidthButton solid $colorWarning;
-        color: $colorWarning;
-        background-color: transparent;
-      }
-    }
-
-    &.critical {
-      color: $colorNeutralWhite;
-      background-color: $colorError;
-
-      &.criticalSecondary {
-        border: $borderWidthButton solid $colorError;
-        color: $colorError;
-        background-color: transparent;
-      }
-    }
-
-    // Kind === dashed
-    &.dashed {
-      color: $colorNeutralDark;
-      background-color: transparent;
-      // Colour set by 'stroke', special characters are encoded
-      background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='%23b7c1cbff' stroke-width='2' stroke-dasharray='6%2c 6' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e");
-
-      &:hover,
-      &.hover,
-      &:focus,
-      &.focus {
-        // Colour set by 'stroke', special characters are encoded
-        background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='%236D7480ff' stroke-width='2' stroke-dasharray='6%2c 6' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e");
-      }
-
-      &:active,
-      &.active {
-        // Colour set by 'stroke', special characters are encoded
-        background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='%23353D49ff' stroke-width='2' stroke-dasharray='6%2c 6' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e");
-        -webkit-box-shadow: inset 0 0 $grid * 2 0 $colorNeutralNormal;
-        -moz-box-shadow: inset 0 0 $grid * 2 0 $colorNeutralNormal;
-        box-shadow: inset 0 0 $grid * 2 0 $colorNeutralNormal;
-      }
-
-      &:disabled {
-        color: $colorNeutralLight;
-        // Colour set by 'stroke', special characters are encoded
-        background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='%23E3E7EDff' stroke-width='2' stroke-dasharray='6%2c 6' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e");
-      }
-    }
-
-    // Kind === danger
-    &.danger {
-      color: $colorNeutralWhite;
-      background-color: $colorError;
-
-      // TODO: Colours not specified
-      &:hover,
-      &.hover,
-      &:focus,
-      &.focus {
-        background-color: pink;
-      }
-
-      &:active,
-      &.active {
-        background-color: $colorError;
-      }
-
-      // TODO: Colours not specified
-      &:disabled {
-        background-color: pink;
-      }
-    }
-  */
+  // Kind === transparent
+  &.transparent {
+    color: c.$colorText;
```

Comment:
Seems like we do not need `color: c.$colorNeutralWhite;` for the `.button` class. If we removed it we could use here `color: inherit` and let the parent container define the color of the button text. It would allow this button to have proper visuals in any parent context. WDYT?

Reply:
Sure, we can do that!